billing.go

```diff
@@ -0,0 +1,165 @@
+package clerk
+
+// Feature represents a feature associated with a plan.
+type Feature struct {
+	APIResource
+
+	Object      string `json:"object"`
+	ID          string `json:"id"`
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	Slug        string `json:"slug"`
+	AvatarURL   string `json:"avatar_url"`
+}
+
+// BillingMoney represents money amounts with formatting.
+type BillingMoney struct {
+	APIResource
+
+	Amount          int64  `json:"amount"`
+	AmountFormatted string `json:"amount_formatted"`
+	Currency        string `json:"currency"`
+	CurrencySymbol  string `json:"currency_symbol"`
+}
+
+// BillingProduct represents a product.
+type BillingProduct struct {
+	APIResource
+
+	Object    string  `json:"object"`
+	ID        string  `json:"id"`
+	Slug      string  `json:"slug"`
+	Currency  string  `json:"currency"`
+	Name      string  `json:"name"`
+	IsDefault bool    `json:"is_default"`
+	Plans     []*Plan `json:"plans"`
+}
+
+// Plan represents a billing plan.
+type Plan struct {
+	APIResource
+
+	Object           string          `json:"object"`
+	ID               string          `json:"id"`
+	Name             string          `json:"name"`
+	Fee              *BillingMoney   `json:"fee"`
+	AnnualMonthlyFee *BillingMoney   `json:"annual_monthly_fee"`
+	AnnualFee        *BillingMoney   `json:"annual_fee"`
+	CurrencySymbol   string          `json:"currency_symbol"`
+	Currency         string          `json:"currency"`
+	Description      string          `json:"description"`
+	ProductID        string          `json:"product_id"`
+	Product          *BillingProduct `json:"product"`
+	IsDefault        bool            `json:"is_default"`
+	IsRecurring      bool            `json:"is_recurring"`
+	PubliclyVisible  bool            `json:"publicly_visible"`
+	HasBaseFee       bool            `json:"has_base_fee"`
+	PayerType        []string        `json:"payer_type"`
+	ForPayerType     string          `json:"for_payer_type"`
+	Slug             string          `json:"slug"`
+	AvatarURL        string          `json:"avatar_url"`
+	Features         []*Feature      `json:"features"`
+	FreeTrialEnabled bool            `json:"free_trial_enabled"`
+	FreeTrialDays    *int            `json:"free_trial_days"`
+}
+
+// PlanList contains a list of plans.
+type PlanList struct {
+	APIResource
+
+	Data       []*Plan `json:"data"`
```

Comment:
```suggestion
	Data       []Plan `json:"data"`
```

We won't return nil plans, right?